File: Feitos em Java/1037.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        double a=entrada.nextDouble();
        if (a>25.00001&&a<=50.0000000){
            System.out.println("Intervalo (25,50]");
        }
        if(a>=0&&a<=25.0000){
            System.out.println("Intervalo [0,25]");
        }
        if(a>50.00000001&&a<=75.0000000){
            System.out.println("Intervalo (50,75]");
        }
        if(a>75.00000001&&a<=100.0000000){
            System.out.println("Intervalo (75,100]");
        }
        if(a<0||a>100.00000001){
            System.out.println("Fora de intervalo");
        }
    }
}

File: Feitos em C++/1037.cpp
#include <iostream>
using namespace std;
int main(){
    double a;
    cin>>a;
    if (a>25.00001&&a<=50.0000000){
        cout<<"Intervalo (25,50]"<<endl;
    }
    if(a>=0&&a<=25.0000){
        cout<<"Intervalo [0,25]"<<endl;
    }
    if(a>50.00000001&&a<=75.0000000){
        cout<<"Intervalo (50,75]"<<endl;
    }
    if(a>75.00000001&&a<=100.0000000){
        cout<<"Intervalo (75,100]"<<endl;
    }
    if(a<0||a>100.00000001){
        cout<<"Fora de intervalo"<<endl;
    }
    return 0;
}
